napari/_app_model/_submenus.py

```diff
@@ -25,6 +25,15 @@
             enablement=LLSCK.active_layer_is_image_3d,
         ),
     ),
+    (
+        MenuId.LAYERLIST_CONTEXT,
+        SubmenuItem(
+            submenu=MenuId.LAYERS_COPY_SPATIAL,
+            title=trans._('Copy Spatial'),
```

Comment:
```suggestion
            title=trans._('Copy scale & transforms'),
```

`Spatial` is rather esoteric to me.